### PR TITLE
Ported argmax/argmin/all/any/prod/mean to ReductionOpInfo

### PR DESCRIPTION
**Pull request by heitorschueroff**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #62920
* #62900
* #62899
* #62737

### CODE REVIEW

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -8479,29 +8457,143 @@ def wrapper(t: torch.Tensor, *args, dim=None, keepdim=False, **kwargs):
...
+            SkipInfo('TestReductions', 'test_result_dtype'),
+            SkipInfo('TestReductions', 'test_ref_random_input_small', dtypes=[torch.uint8]),
+            # FIXME: out is not support when calling without dim and keepdim
+            SkipInfo('TestCommon', 'test_out'),
+        ),
+    ),
+    ReductionOpInfo(
+        'any',
+        identity=False,
+        nan_policy='omit',
```
Comment (heitorschueroff) [inline torch/testing/_internal/common_methods_invocations.py:8507]: This is incorrect! all/any treat np.nan as True. Maybe add a policy 'ignore' that does not do anything special with np.nan and just passes it through to the operation?

Comment (heitorschueroff) [reply torch/testing/_internal/common_methods_invocations.py:8507]: Or a policy 'largest' that treats NaN as the largest value, which also means non-zero and thus True? This would be inline with our sorting routine which sorts NaN as the largest value.

File: test/test_reductions.py [hunk 2]
```diff
@@ -179,34 +179,52 @@ def test_dim_offbounds_keepdim(self, device, dtype, op: ReductionOpInfo):
...
     @ops(reduction_op_db, allowed_dtypes=[torch.float])
     def test_non_empty_slice_of_empty_tensor(self, device, dtype, op: ReductionOpInfo):
         """Tests that reducing a nonzero size dimension of an empty tensor is
         allowed and returns an empty tensor with the dimensions reduced
         """
         t = make_tensor((0, 2, 3), device, dtype)
-        for dim in [1] + [[1, 2]] if op.supports_multiple_dims else []:
+        for dim, keepdim in product([1, [1, 2]], [True, False]):
+            if isinstance(dim, Sequence) and not op.supports_multiple_dims:
+                continue
```
Comment (mruberry) [inline test/test_reductions.py:222]: Comment for this skip

File: test/test_reductions.py [hunk 3]
```diff
@@ -179,34 +179,52 @@ def test_dim_offbounds_keepdim(self, device, dtype, op: ReductionOpInfo):
...
+        test_cases = (
+            ([0], [0]),
+            ([2, 0, 3], [1, [0, 1], [1, 2]]),
+        )
+
+        for shape, dims in test_cases:
+            t = make_tensor(shape, device, dtype)
+
+            for dim, keepdim in product(dims, [True, False]):
+                if isinstance(dim, Sequence) and not op.supports_multiple_dims:
```
Comment (mruberry) [inline test/test_reductions.py:193]: Comment for this skip

File: test/test_reductions.py [hunk 4]
```diff
@@ -179,34 +179,52 @@ def test_dim_offbounds_keepdim(self, device, dtype, op: ReductionOpInfo):
...
+                    continue
+
+                args, kwargs = next(op.generate_args_kwargs(t, dim=dim))
```
Comment (mruberry) [inline test/test_reductions.py:196]: Comment for only testing on first args/kwargs

File: torch/testing/_internal/common_methods_invocations.py [hunk 5]
```diff
@@ -762,7 +762,7 @@ def default_test_dtypes(self, device_type):
 def _generate_reduction_inputs(device, dtype, requires_grad):
     yield make_tensor((), device, dtype, requires_grad=requires_grad)
     yield make_tensor((2,), device, dtype, requires_grad=requires_grad)
-    yield make_tensor((2, 3), device, dtype, requires_grad=requires_grad, noncontiguous=True)
+    yield make_tensor((2, 3), device, dtype, requires_grad=requires_grad)
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:765]: Why this change?

File: torch/testing/_internal/common_methods_invocations.py [hunk 6]
```diff
@@ -855,9 +855,10 @@ def __init__(
         identity: Optional[Union[int, float, complex]] = None,
 
         # possible values are:
-        # - propagate: NaN values are propagated to the output
-        # - omit: only non-NaN values participate in computing reduction
-        nan_policy: str = 'propagate',
+        # - 'propagate': NaN values are propagated to the output
+        # - 'omit': only non-NaN values participate in computing reduction
+        # - None: operator decides how to handle NaNs.
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:860]: Would you elaborate on this addition?

File: torch/testing/_internal/common_methods_invocations.py [hunk 7]
```diff
@@ -8479,21 +8458,113 @@ def wrapper(t: torch.Tensor, *args, dim=None, keepdim=False, **kwargs):
...
+        result_dtype=torch.bool,
+        dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+        reference=reference_reduction_numpy(np.all),
+        skips=(
+            # FIXME: does not support passing keepdim without dim
+            SkipInfo('TestReductions', 'test_dim_default_keepdim'),
+            # FIXME: does not support dim=None
+            SkipInfo('TestReductions', 'test_dim_none'),
+            SkipInfo('TestReductions', 'test_dim_none_keepdim'),
+            # FIXME: uint8 input returns uint8 instead of bool
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:8499]: There may be an issue for this

File: torch/testing/_internal/common_methods_invocations.py [hunk 8]
```diff
@@ -8479,21 +8458,113 @@ def wrapper(t: torch.Tensor, *args, dim=None, keepdim=False, **kwargs):
...
+            SkipInfo('TestReductions', 'test_result_dtype'),
+            SkipInfo('TestReductions', 'test_ref_random_input_small', dtypes=[torch.uint8]),
+            # FIXME: out is not support when calling without dim and keepdim
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:8502]: Super frustrating

File: torch/testing/_internal/common_methods_invocations.py [hunk 9]
```diff
@@ -8504,6 +8575,32 @@ def wrapper(t: torch.Tensor, *args, dim=None, keepdim=False, **kwargs):
...
+        skips=(
+            # FIXME: does not support passing keepdim without passing dim
+            SkipInfo('TestReductions', 'test_dim_default_keepdim'),
+            # FIXME: sum reduces all dimensions when dim=[]
+            SkipInfo('TestReductions', 'test_dim_empty'),
+            SkipInfo('TestReductions', 'test_dim_empty_keepdim'),
+            # FIXME: sum does not support passing None to dim
+            SkipInfo('TestReductions', 'test_dim_none'),
+            SkipInfo('TestReductions', 'test_dim_none_keepdim'),
+            # FIXME: failed to compare as equal with error 0.000102996826171875
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:8599]: Nice thorough comments with clear statements of the issues

File: torch/testing/_internal/common_methods_invocations.py (hunk 6 above)
Comment (mruberry) [reply torch/testing/_internal/common_methods_invocations.py:860]: None isn't actually used by any of the ReductionInfos, right?

**Review by mruberry — COMMENTED** [review]
The additional ReductionInfos look great. It might make sense to move them earlier in this stack (unless that's a pain). A few questions/comments inline but overall looks very good (like the rest of this stack)

File: torch/testing/_internal/common_methods_invocations.py (hunk 6 above)
Comment (heitorschueroff) [reply torch/testing/_internal/common_methods_invocations.py]: Though they could still offer nan_policy if a user for instance wants to compute all of only the non-NaN values.

File: torch/testing/_internal/common_methods_invocations.py (hunk 9 above)
Comment (heitorschueroff) [reply torch/testing/_internal/common_methods_invocations.py]: I moved the precision problems into a DecorateInfo with toleranceOverride to make a distinction between wrong result and stability issues.

File: torch/testing/_internal/common_methods_invocations.py (hunk 7 above)
Comment (heitorschueroff) [reply torch/testing/_internal/common_methods_invocations.py:8580]: Yup, many of these skips have issues opened already. I was lazy to add links to all of them since they're all tracked in the reduction tracking issue now.

File: torch/testing/_internal/common_methods_invocations.py [hunk 10]
```diff
@@ -8770,6 +8685,76 @@ def wrapper(x: np.ndarray, *args, **kwargs):
...
+        result_dtype=torch.int64,
+        dtypes=all_types_and(torch.float16, torch.bfloat16),
+        ref=reference_reduction_numpy(np.argmax, supports_keepdims=False),
+        skips=(
+            # FIXME: keepdim parameter is ignored when dim=None
+            SkipInfo('TestReductions', 'test_dim_default_keepdim'),
+            SkipInfo('TestReductions', 'test_dim_none_keepdim'),
+        ),
+    ),
+    ReductionOpInfo(
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:8701]: Maybe we should write a test that enforces an alphabetical order...

Comment (mruberry) [reply torch/testing/_internal/common_methods_invocations.py:8701]: (But not seriously because we might break up `op_db` into smaller pieces because it's growing too big)

**Comment by mruberry** [comment]
This test failure is real:

```
======================================================================
FAIL [0.000s]: test_ref_small_input_prod_cpu_uint8 (__main__.TestReductionsCPU)
Compares op against reference for many small random input tensors
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\actions-runner\_work\pytorch\pytorch\pytorch-1140309696\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 373, in instantiated_test
    result = test(self, **param_kwargs)
  File "C:\actions-runner\_work\pytorch\pytorch\pytorch-1140309696\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 766, in test_wrapper
    return test(*args, **kwargs)
  File "test_reductions.py", line 362, in test_ref_small_input
    self._test_ref(op, make(32))
  File "test_reductions.py", line 347, in _test_ref
    self.assertEqual(result, expected, exact_dtype=False)
  File "C:\actions-runner\_work\pytorch\pytorch\pytorch-1140309696\build\win_tmp\build\torch\testing\_internal\common_utils.py", line 1463, in assertEqual
    self.assertEqual(x.item(), y, atol=atol, rtol=rtol, msg=msg,
  File "C:\actions-runner\_work\pytorch\pytorch\pytorch-1140309696\build\win_tmp\build\torch\testing\_internal\common_utils.py", line 1632, in assertEqual
    super().assertTrue(result, msg=self._get_assert_msg(msg, debug_msg=debug_msg))
AssertionError: False is not true : Scalars failed to compare as equal! 8360415632944005120 != 138412032
```

as is this one:

```
Aug 17 19:14:57   test_noncontiguous_expanded_prod_cpu_int64 (__main__.TestReductionsCPU)
Aug 17 19:14:57 Tests reducing a tensor with expanded singleton dimensions. ... /var/lib/jenkins/workspace/build/aten/src/ATen/native/cpu/ReduceOpsKernel.cpp.AVX2.cpp:176:5: runtime error: signed integer overflow: -7524374069649604608 * 7 cannot be represented in type 'long'
Aug 17 19:14:57 ==394==WARNING: invalid path to external symbolizer!
Aug 17 19:14:57 ==394==WARNING: Failed to use and restart external symbolizer!
Aug 17 19:14:57     #0 0x7f3388fede3f  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x1b48fe3f)
Aug 17 19:14:57     #1 0x7f3388fed9e9  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x1b48f9e9)
Aug 17 19:14:57     #2 0x7f3388febf4b  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x1b48df4b)
Aug 17 19:14:57     #3 0x7f3388feae1d  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x1b48ce1d)
Aug 17 19:14:57     #4 0x7f3388fea6de  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x1b48c6de)
Aug 17 19:14:57     #5 0x7f337cb5a715  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0xeffc715)
Aug 17 19:14:57     #6 0x7f337cb41ce7  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0xefe3ce7)
Aug 17 19:14:57     #7 0x7f3388fea15c  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x1b48c15c)
Aug 17 19:14:57     #8 0x7f3388fca4c9  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x1b46c4c9)
Aug 17 19:14:57     #9 0x7f337d3d9aed  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0xf87baed)
Aug 17 19:14:57     #10 0x7f337d3da325  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0xf87c325)
Aug 17 19:14:57     #11 0x7f337ef88e24  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x1142ae24)
Aug 17 19:14:57     #12 0x7f337e1aac2a  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x1064cc2a)
Aug 17 19:14:57     #13 0x7f337eb45388  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x10fe7388)
Aug 17 19:14:57     #14 0x7f3384e514f5  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x172f34f5)
Aug 17 19:14:57     #15 0x7f3384e50938  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x172f2938)
Aug 17 19:14:57     #16 0x7f337eb44d30  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_cpu.so+0x10fe6d30)
Aug 17 19:14:57     #17 0x7f33a8531b6c  (/opt/conda/lib/python3.6/site-packages/torch/lib/libtorch_python.so+0x3d0fb6c)
Aug 17 19:14:57     #18 0x564ff304d665  (/opt/conda/bin/python3.6+0x157665)
Aug 17 19:14:57     #19 0x564ff3097e57  (/opt/conda/bin/python3.6+0x1a1e57)
Aug 17 19:14:57     #20 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #21 0x564ff306937b  (/opt/conda/bin/python3.6+0x17337b)
Aug 17 19:14:57     #22 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #23 0x564ff306f90b  (/opt/conda/bin/python3.6+0x17990b)
Aug 17 19:14:57     #24 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #25 0x564ff309ab76  (/opt/conda/bin/python3.6+0x1a4b76)
Aug 17 19:14:57     #26 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #27 0x564ff306341a  (/opt/conda/bin/python3.6+0x16d41a)
Aug 17 19:14:57     #28 0x564ff30790f6  (/opt/conda/bin/python3.6+0x1830f6)
Aug 17 19:14:57     #29 0x564ff309639e  (/opt/conda/bin/python3.6+0x1a039e)
Aug 17 19:14:57     #30 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #31 0x564ff3068b64  (/opt/conda/bin/python3.6+0x172b64)
Aug 17 19:14:57     #32 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #33 0x564ff309ab76  (/opt/conda/bin/python3.6+0x1a4b76)
Aug 17 19:14:57     #34 0x564ff3026993  (/opt/conda/bin/python3.6+0x130993)
Aug 17 19:14:57     #35 0x564ff3068b64  (/opt/conda/bin/python3.6+0x172b64)
Aug 17 19:14:57     #36 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #37 0x564ff309ab76  (/opt/conda/bin/python3.6+0x1a4b76)
Aug 17 19:14:57     #38 0x564ff3026993  (/opt/conda/bin/python3.6+0x130993)
Aug 17 19:14:57     #39 0x564ff306341a  (/opt/conda/bin/python3.6+0x16d41a)
Aug 17 19:14:57     #40 0x564ff30790f6  (/opt/conda/bin/python3.6+0x1830f6)
Aug 17 19:14:57     #41 0x564ff309639e  (/opt/conda/bin/python3.6+0x1a039e)
Aug 17 19:14:57     #42 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #43 0x564ff306341a  (/opt/conda/bin/python3.6+0x16d41a)
Aug 17 19:14:57     #44 0x564ff30790f6  (/opt/conda/bin/python3.6+0x1830f6)
Aug 17 19:14:57     #45 0x564ff3097a1d  (/opt/conda/bin/python3.6+0x1a1a1d)
Aug 17 19:14:57     #46 0x564ff3026993  (/opt/conda/bin/python3.6+0x130993)
Aug 17 19:14:57     #47 0x564ff306341a  (/opt/conda/bin/python3.6+0x16d41a)
Aug 17 19:14:57     #48 0x564ff30790f6  (/opt/conda/bin/python3.6+0x1830f6)
Aug 17 19:14:57     #49 0x564ff3097a1d  (/opt/conda/bin/python3.6+0x1a1a1d)
Aug 17 19:14:57     #50 0x564ff3026993  (/opt/conda/bin/python3.6+0x130993)
Aug 17 19:14:57     #51 0x564ff306937b  (/opt/conda/bin/python3.6+0x17337b)
Aug 17 19:14:57     #52 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #53 0x564ff309ab76  (/opt/conda/bin/python3.6+0x1a4b76)
Aug 17 19:14:57     #54 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #55 0x564ff3069160  (/opt/conda/bin/python3.6+0x173160)
Aug 17 19:14:57     #56 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #57 0x564ff306f90b  (/opt/conda/bin/python3.6+0x17990b)
Aug 17 19:14:57     #58 0x564ff302728b  (/opt/conda/bin/python3.6+0x13128b)
Aug 17 19:14:57     #59 0x564ff3079134  (/opt/conda/bin/python3.6+0x183134)
Aug 17 19:14:57     #60 0x564ff309639e  (/opt/conda/bin/python3.6+0x1a039e)
Aug 17 19:14:57     #61 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #62 0x564ff306937b  (/opt/conda/bin/python3.6+0x17337b)
Aug 17 19:14:57     #63 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #64 0x564ff309ab76  (/opt/conda/bin/python3.6+0x1a4b76)
Aug 17 19:14:57     #65 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #66 0x564ff3069160  (/opt/conda/bin/python3.6+0x173160)
Aug 17 19:14:57     #67 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #68 0x564ff306f90b  (/opt/conda/bin/python3.6+0x17990b)
Aug 17 19:14:57     #69 0x564ff302728b  (/opt/conda/bin/python3.6+0x13128b)
Aug 17 19:14:57     #70 0x564ff3079134  (/opt/conda/bin/python3.6+0x183134)
Aug 17 19:14:57     #71 0x564ff309639e  (/opt/conda/bin/python3.6+0x1a039e)
Aug 17 19:14:57     #72 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #73 0x564ff306937b  (/opt/conda/bin/python3.6+0x17337b)
Aug 17 19:14:57     #74 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #75 0x564ff309ab76  (/opt/conda/bin/python3.6+0x1a4b76)
Aug 17 19:14:57     #76 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #77 0x564ff3069160  (/opt/conda/bin/python3.6+0x173160)
Aug 17 19:14:57     #78 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #79 0x564ff306f90b  (/opt/conda/bin/python3.6+0x17990b)
Aug 17 19:14:57     #80 0x564ff302728b  (/opt/conda/bin/python3.6+0x13128b)
Aug 17 19:14:57     #81 0x564ff3079134  (/opt/conda/bin/python3.6+0x183134)
Aug 17 19:14:57     #82 0x564ff309639e  (/opt/conda/bin/python3.6+0x1a039e)
Aug 17 19:14:57     #83 0x564ff3062ec3  (/opt/conda/bin/python3.6+0x16cec3)
Aug 17 19:14:57     #84 0x564ff30790f6  (/opt/conda/bin/python3.6+0x1830f6)
Aug 17 19:14:57     #85 0x564ff309639e  (/opt/conda/bin/python3.6+0x1a039e)
Aug 17 19:14:57     #86 0x564ff3062ec3  (/opt/conda/bin/python3.6+0x16cec3)
Aug 17 19:14:57     #87 0x564ff30790f6  (/opt/conda/bin/python3.6+0x1830f6)
Aug 17 19:14:57     #88 0x564ff309639e  (/opt/conda/bin/python3.6+0x1a039e)
Aug 17 19:14:57     #89 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #90 0x564ff306937b  (/opt/conda/bin/python3.6+0x17337b)
Aug 17 19:14:57     #91 0x564ff30289fd  (/opt/conda/bin/python3.6+0x1329fd)
Aug 17 19:14:57     #92 0x564ff306be8f  (/opt/conda/bin/python3.6+0x175e8f)
Aug 17 19:14:57     #93 0x564ff304d7d6  (/opt/conda/bin/python3.6+0x1577d6)
Aug 17 19:14:57     #94 0x564ff302728b  (/opt/conda/bin/python3.6+0x13128b)
Aug 17 19:14:57     #95 0x564ff3063c16  (/opt/conda/bin/python3.6+0x16dc16)
Aug 17 19:14:57     #96 0x564ff3079134  (/opt/conda/bin/python3.6+0x183134)
Aug 17 19:14:57     #97 0x564ff3097a1d  (/opt/conda/bin/python3.6+0x1a1a1d)
Aug 17 19:14:57     #98 0x564ff30bc574  (/opt/conda/bin/python3.6+0x1c6574)
Aug 17 19:14:57     #99 0x564ff30790f6  (/opt/conda/bin/python3.6+0x1830f6)
Aug 17 19:14:57     #100 0x564ff309639e  (/opt/conda/bin/python3.6+0x1a039e)
Aug 17 19:14:57     #101 0x564ff302615f  (/opt/conda/bin/python3.6+0x13015f)
Aug 17 19:14:57     #102 0x564ff30688f2  (/opt/conda/bin/python3.6+0x1728f2)
Aug 17 19:14:57     #103 0x564ff30d0cd5  (/opt/conda/bin/python3.6+0x1dacd5)
Aug 17 19:14:57     #104 0x564ff30d31cd  (/opt/conda/bin/python3.6+0x1dd1cd)
Aug 17 19:14:57     #105 0x564ff30d33ba  (/opt/conda/bin/python3.6+0x1dd3ba)
Aug 17 19:14:57     #106 0x564ff30d3b43  (/opt/conda/bin/python3.6+0x1ddb43)
Aug 17 19:14:57     #107 0x564ff300d196  (/opt/conda/bin/python3.6+0x117196)
Aug 17 19:14:57     #108 0x7f33b0f8d83f  (/lib/x86_64-linux-gnu/libc.so.6+0x2083f)
Aug 17 19:14:57     #109 0x564ff309d33d  (/opt/conda/bin/python3.6+0x1a733d)
Aug 17 19:14:57 
Aug 17 19:14:57 SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior /var/lib/jenkins/workspace/build/aten/src/ATen/native/cpu/ReduceOpsKernel.cpp.AVX2.cpp:176:5 in 
Aug 17 19:14:57 Traceback (most recent call last):
Aug 17 19:14:57   File "test/run_test.py", line 1095, in <module>
Aug 17 19:14:57     main()
Aug 17 19:14:57   File "test/run_test.py", line 1074, in main
Aug 17 19:14:57     raise RuntimeError(err_message)
Aug 17 19:14:57 RuntimeError: test_reductions failed!
```

**Comment by heitorschueroff** [comment]
I moved these ports down the stack
